openedx/course-discovery
# feat: filter restricted runs on APIs

Internal tickets: [PROD-4005](https://2u-internal.atlassian.net/browse/PROD-4005) + [PROD-4006](https://2u-internal.atlassian.net/browse/PROD-4006)

Hide restricted runs across APIs by default. Also do not include them when calculating fields like course_run_statuses. If a query-param `include_restricted=<some_value>` is present, do not hide them

# Testing
1. Create restricted runs in your system and link them with courses, programs, etc.
2. Run `update_index`
3. Verify that runs are hidden on all APIs by default and are revealed on adding the query params.
